code/game/gamemodes/brother/traitor_bro.dm

```diff
@@ -24,6 +24,8 @@
 		restricted_jobs += protected_jobs
 	if(CONFIG_GET(flag/protect_assistant_from_antagonist))
 		restricted_jobs += "Assistant"
+	if(CONFIG_GET(flag/protect_heads_from_antagonist))

```

Comment:
Does this really have to be done with each individual game mode?

Reply:
Yea, like the other 2 flags